### PR TITLE
Added : The sample web application project for claims-based authentication of OpenID Connect by OWIN.

### LINKED ISSUE #12: Research to support claims-based authentication of OpenID Connect by OWIN security components
# Requirement
To support OpenID Connect with Microsoft.Owin.Security.OpenIdConnect in OpenTouryo, please add the sample code [here](https://github.com/OpenTouryoProject/SampleProgram/tree/master/ASPNET/OpenID Connect).
- This sample is "preliminary investigation" "to support".
- This sample does not need to use the OpenToutryo.

##### As well as the following.
- Research to support claims-based authentication of ID-Federation by WIF #7

##### Reference
- IdM実験室 WIF Extension for OAuth を使って OpenID Connect を体験  
http://idmlab.eidentity.jp/2012/03/wif-extension-for-oauth-openid-connect.html

### PR DESCRIPTION
**Pull request by daisukenishino**
fixed #12

### CODE REVIEW

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Operation check was already performed.
Please comfirm.

File: ASPNET/OpenID_Connect/OpenIDConnect_sample/OpenIDConnect_sample/Default.aspx.cs [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+                // Send an OpenID Connect sign-in request.
+                Context.GetOwinContext().Authentication.Challenge(new AuthenticationProperties { RedirectUri = "/" }, OpenIdConnectAuthenticationDefaults.AuthenticationType);
+            }
+            else
+            {
+                // Assigning user Identity name to the label
+                this.lblLoggedInUser.Text = Context.User.Identity.Name;
+
+                // To do
+                // ...
```
Comment (yusukemaegawa) [inline ASPNET/OpenID_Connect/OpenIDConnect_sample/OpenIDConnect_sample/Default.aspx.cs]: @daisukenishino 

What are you saying that developers should do?

**Comment by yusukemaegawa** [comment]
@daisukenishino 

> https://github.com/OpenTouryoProject/SampleProgram/issues/12

Please present the relationship between this sample and OpenTouryo.
(Like #7)

**Comment by daisukenishino** [comment]
@yusukemaegawa 

>https://github.com/OpenTouryoProject/SampleProgram/pull/14#issuecomment-185996889

Please check that it has been added the description at the beginning.
https://github.com/OpenTouryoProject/SampleProgram/issues/12#issue-129066709

**Comment by daisukenishino** [comment]
@yusukemaegawa 

Your pointed out has been reflected.
https://github.com/OpenTouryoProject/SampleProgram/pull/14/files#diff-ad4ca12bfe935c6230174665b5df9174R69

**Comment by yusukemaegawa** [comment]
@daisukenishino 

```c#
// TODO: Coding the logic after the authentication process has been completed.
// ...
```

I do not understand whether "you intend to write code" or "you want the user of this sample to write code". If you want the user of this sample to write code, I think you should specify so.

e.g.
```c#
// TODO: Please implement the processing after the authentication.
// ...
```

**Comment by daisukenishino** [comment]
@yusukemaegawa 

>https://github.com/OpenTouryoProject/SampleProgram/pull/14#issuecomment-186019444

Your pointed out has been reflected.
https://github.com/OpenTouryoProject/SampleProgram/pull/14/files#diff-ad4ca12bfe935c6230174665b5df9174R69

**Pull request merged**